eureka-client/src/main/java/com/netflix/discovery/converters/EurekaJacksonCodec.java

```diff
@@ -50,7 +50,7 @@
 
     private static final Logger logger = LoggerFactory.getLogger(EurekaJacksonCodec.class);
 
-    private static final Version VERSION = new Version(1, 1, 0, null);
+    protected static final Version VERSION = new Version(1, 1, 0, null);
```

Comment:
Is this refactoring needed to be able to extend serializer to support other data center types?